db/migrate/20210802151907_add_dredging_length_to_transient_registration.rb

```diff
@@ -0,0 +1,5 @@
+class AddDredgingLengthToTransientRegistration < ActiveRecord::Migration[6.0]
+  def change
+    add_column :transient_registrations, :dredging_length, :string
```

Comment:
This is validated as an integer. Could it be an integer column also?

Reply:
Yeah, for some reason it was a string in the existing records but I really don't see why! I'll change it 👍